Mathlib/AlgebraicGeometry/EllipticCurve/Affine.lean

```diff
@@ -770,180 +775,252 @@ section BaseChange
 
 /-! ### Maps and base changes -/
 
-variable {A : Type v} [CommRing A] (φ : R →+* A)
```

Comment:
Just wondering, is there any "real" reason to rename all the $\varphi$ to $f$?

Reply:
It doesn't make a difference in this file, but I plan to use `f` for all maps going forward (e.g. in the upcoming `Projective`, `Jacobian`, and `DivisionPolynomial` files), because `\phi` and `\psi` will be reserved for division polynomials.